partiql-tests-data/eval/primitives/basic.ion

```diff
@@ -310,7 +310,7 @@ basic::[
       output:$bag::[
         0,
         1,
-        1,
+        2,
```

Comment:
Could you move this test and the tests adjusted in `cast.ion` to [partiql-tests-data-extended](https://github.com/partiql/partiql-tests/tree/ab0916c98cc0f1033dc8a41c68434c3f079ab51b/partiql-tests-data-extended)? As part of https://github.com/partiql/partiql-tests/issues/39, we'll define the numeric modeling semantics (including rounding behavior) in the conformance tests.